roles/cadvisor/tasks/main.yml

```diff
@@ -28,13 +25,13 @@
   tags:
     - cadvisor
 
-#- name: Set cadvisor consul service definition
-#  sudo: yes
-#  template:
-#   src: cadvisor-consul.j2
-#    dest: "{{ cadvisor_consul_dir }}/cadvisor.json"
-#  notify:
-#    - restart consul
-#  when: cadvisor_enabled
-#  tags:
-#    - cadvisor
+- name: Set cadvisor consul service definition
```

Comment:
I've added a better cadvisor health check in https://github.com/Capgemini/Apollo/pull/598/files might want to combine these PRs